src/openrct2/config/ConfigEnum.h

```diff
@@ -0,0 +1,79 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
There is `ConfigEnum` implementation in this class, this is why I asked about it not being `.hpp`